users/management/commands/merge_users.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from sentry_sdk import capture_exception
+
+from users.utils import merge_users
+from users.models import User
+
+
+class Command(BaseCommand):
+    help = (
+        'Merge two users into one.'
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--live-run',
+            dest='live_run',
+            action='store_true',
+            help='opposite of dry run - will actually write data to the database',
+            required=False,
+        )
+        parser.add_argument(
+            '--from-email',
+            dest='from_email',
+            help='The email of the user you want to merge into the other',
```

Comment:
Should we delete the old user? If so add "this user will be deleted"

Reply:
I think we should. I'll add that message.